uber/config.py

```diff
@@ -592,6 +592,15 @@ def _is_intstr(s):
         c.TRANSFERABLE_BADGE_TYPES.append(_val)
         _day += timedelta(days=1)
 
+"""
```

Comment:
This was the hackiest thing I had to do. The alternative was to put `str()` around every single ribbon variable in the code. That seemed like a bad idea.